plugin/trino-hive/pom.xml

```diff
@@ -255,6 +255,11 @@
             <artifactId>alluxio-shaded-client</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.iceberg</groupId>
+            <artifactId>iceberg-api</artifactId>
+        </dependency>
```

Comment:
Let's not bring iceberg dependency into hive connector

Reply:
yeah, this will be removed!